content/en/blogs/install-kubernetes-containerd-multus.md

```diff
@@ -0,0 +1,259 @@
+---
+title: 'Install Kubernetes 1.23, containerd, and Multus CNI the Easy Way'
```

Comment:
Maybe "1.23.0" would be better?

Reply:
As I see `1.23` is commonly used in the [Kubernetes official blog](https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/), I think we don't need to emphasize `1.23.0`.